src/pages/Loans/LoansOverview/components/LoanDetails/LoanDetails.tsx

```diff
@@ -0,0 +1,47 @@
+import { LoanAsset } from '@interlay/interbtc-api';
+
+import { TransactionDetails, TransactionDetailsDd, TransactionDetailsDt, TransactionDetailsGroup } from '@/components';
+import { LoanAction } from '@/types/loans';
+import { Prices } from '@/utils/hooks/api/use-get-prices';
+
+import { getApyLabel } from '../../utils/apy';
+import { RewardsDetails } from './RewardsDetails';
+
+type LoanDetailsProps = {
+  variant?: LoanAction;
+  asset?: LoanAsset;
+  prices?: Prices;
+};
+
+const LoanDetails = ({ variant, asset, prices }: LoanDetailsProps): JSX.Element | null => {
+  const isBorrow = variant === 'borrow';
+  const apy = isBorrow ? asset?.borrowApy : asset?.lendApy;
+
+  if (!apy || !asset) {
+    return null;
+  }
+
+  const rewards = isBorrow ? asset?.borrowReward : asset?.lendReward;
+
+  return (
+    <TransactionDetails>
+      <TransactionDetailsGroup>
+        <TransactionDetailsDt>
+          {isBorrow ? 'Borrow' : 'Lend'} APY {asset.currency.ticker}
+        </TransactionDetailsDt>
+        <TransactionDetailsDd>{getApyLabel(apy)}</TransactionDetailsDd>
+      </TransactionDetailsGroup>
```

Comment:
Just a thought on naming of these components, I think we should rename it to something more clear, because in this case it has nothing to do with the transaction but a mere display of data, what do you think?

Reply:
Im open to suggestions.

Naming is hard :D